docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -70,13 +65,100 @@ represented in OpenVINO IR using NNCF:
          :fragment: [compression_8bit]
 
 
-Now, the model is ready for compilation and inference.
-It can be also saved into a compressed format, resulting in a smaller binary file.
+Now, the model is ready for compilation and inference. See the example below,
+which shows data-free 8-bit weight quantization of OpenVINO IR.
+
+Before you run the code sample, make sure Optimum Intel
+is installed in your environment by running the following command:
+
+.. code-block:: python
+
+   pip install optimum[openvino]
+
+If the model comes from `Hugging Face <https://huggingface.co/models>`__ and is supported
+by Optimum, it may be easier to use the **Optimum Intel API**, which employs NNCF weight
+compression capabilities to optimize various large Transformer models.
+
+The NNCF ``nncf.compress_weights()`` API, with most of its options, is exposed in the
+``.from_pretrained()`` method of Optimum Intel classes. Optimum also has several datasets
+for data-aware quantization available out-of-the-box.
+
+.. tab-set::
+
+   .. tab-item:: Compression with NNCF
+      :sync: nncf
+
+      Load a pre-trained Hugging Face model, using the Optimum Intel API,
+      compress it to INT8_ASYM, using NNCF, and then execute inference with a text phrase:
+
+      .. code-block:: python
+
+        from nncf import compress_weights, CompressWeightsMode
+        from optimum.intel.openvino import OVModelForCausalLM
+        from transformers import AutoTokenizer, pipeline
+
+        # Load a model and compress it with NNCF.
+        model_id = "microsoft/Phi-3.5-mini-instruct"
+        model = OVModelForCausalLM.from_pretrained(model_id, export=True, load_in_8bit=False, compile=False)
+        model.model = compress_weights(model.model, mode=CompressWeightsMode.INT8_ASYM)
+
+        # Inference
+        model.compile()
+        tokenizer = AutoTokenizer.from_pretrained(model_id)
+        pipe = pipeline("text-generation", model=model, tokenizer=tokenizer)
+        phrase = "The weather is"
+        results = pipe(phrase)
+        print(results)
+
+   .. tab-item:: Compression with Optimum-Intel
+      :sync: optimum
+
+      Load a pre-trained Hugging Face model, compress it to INT8_ASYM, using the
+      Optimum Intel API, and then execute inference with a text phrase:
+
+      .. code-block:: python
+
+         from optimum.intel.openvino import OVModelForCausalLM, OVWeightQuantizationConfig
+         from transformers import AutoTokenizer, pipeline
+
+         # Load and compress a model from Hugging Face.
+         model_id = "microsoft/Phi-3.5-mini-instruct"
+         model = OVModelForCausalLM.from_pretrained(
+             model_id,
+             export=True,
+             quantization_config=OVWeightQuantizationConfig(
+                 bits=8,
+                 quant_method="awq",
```

Comment:
This is a config for int4 quantization (not int8) and it should be moved below with the change of `bits=4` and the comment above.
Also you have the command-line variant of it which is for int4 as you can see `optimum-cli export openvino --model microsoft/Phi-3.5-mini-instruct --weight-format int4 --awq --scale-estimation --dataset wikitext2 --group-size 64 --ratio 1.0 ov_phi-3.5-mini-instruct`.

For INT8 it looks like:
```
quantization_config=OVWeightQuantizationConfig(bits=8)
```
and
```
optimum-cli export openvino --model microsoft/Phi-3.5-mini-instruct --weight-format int8 ov_phi-3.5-mini-instruct
```

Reply:
I have applied necessary changes in: https://github.com/openvinotoolkit/openvino/pull/27432/commits/5b49f023635c0ac453d5ead14e57f4fc149407c3